src/main/java/com/adyen/Client.java

```diff
@@ -92,17 +92,13 @@ public Client(String username, String password, Environment environment, String
      * Use this constructor to create client for client certificate authentication along with API key.
      * Note: Client certificate authentication is only applicable for PAL and Checkout services in LIVE,
      * Other services will just use API key for authentication.
-     * @param trustStore Trust store containing server certificate
-     * @param clientKeyStore Client Key store containing client certificate and key
-     * @param clientKeyStorePassword Password for client key store
+     * @param sslContext {@link SSLContext} for client certificate authentication
      * @param apiKey Adyen API Key
      * @param region Data center region (EU/US/AU), default EU if not provided
      */
-    public Client(KeyStore trustStore, KeyStore clientKeyStore, String clientKeyStorePassword, String apiKey, Region region) {
+    public Client(SSLContext sslContext, String apiKey, Region region) {
         this(apiKey, Environment.LIVE);
-        this.config.setClientKeyStorePassword(clientKeyStorePassword);
-        this.config.setClientKeyStore(clientKeyStore);
-        this.config.setTrustKeyStore(trustStore);
+        this.config.setSSLContext(sslContext);
         this.config.setEndpoint(ENDPOINT_CERT_LIVE);
```

Comment:
This endpoint in the config actually does not get used for anything. Does this get used for something or?